code/modules/projectiles/guns/energy/energy_gun.dm

```diff
@@ -150,3 +150,80 @@
 				add_overlay("[icon_state]_fail_1")
 			if(151 to INFINITY)
 				add_overlay("[icon_state]_fail_2")
+
+/obj/item/gun/energy/e_gun/large //SPECIAL THANKS to Jake Park for helping me with the alert lock code! You're the best, man <3
```

Comment:
Remove this gun, or atomize it into its own PR, then I'll be in favor for merging the PR

Reply:
No.

Reply:
More specifically, No for reason I stated in #Dev-Main on the discord.